templates/typescript_gapic/package.json.njk

```diff
@@ -5,6 +5,11 @@
   "author": "Google LLC",
   "description": "{{ api.naming.productName }} client for Node.js",
   "main": "build/src/index.js",
+  "files": [
+    "build/src",
+    "build/protos",
+    "!build/src/**/*.map"
```

Comment:
Can we re-litigate this :smile: I think having the maps is good, especially now that I've added source-map-support to Node.js.